base/serialize.jl

```diff
@@ -437,7 +437,9 @@ function serialize(s::AbstractSerializer, t::Task)
 end
 
 function serialize(s::AbstractSerializer, g::GlobalRef)
-    if g.mod === Main && isdefined(g.mod, g.name) && isconst(g.mod, g.name)
+    if (g.mod === __deserialized_types__ ) ||
+        (g.mod === Main && isdefined(g.mod, g.name) && isconst(g.mod, g.name))
```

Comment:
superficial, but I think this would look a little nicer as
```diff
diff --git a/base/distributed/clusterserialize.jl b/base/distributed/clusterserialize.jl
index 9fb256d1e2..a5ed2cca61 100644
--- a/base/distributed/clusterserialize.jl
+++ b/base/distributed/clusterserialize.jl
@@ -89,8 +89,7 @@ function serialize(s::ClusterSerializer, g::GlobalRef)
     sym = g.name
     if g.mod === Main && isdefined(g.mod, sym)
         if (binding_module(Main, sym) === Main) && (s.anonfunc_id != 0) &&
-            !startswith(string(sym), "#") # Anonymous functions are handled via FULL_GLOBALREF_TAG
-
+                !startswith(string(sym), "#") # Anonymous functions are handled via FULL_GLOBALREF_TAG
             push!(get!(s.glbs_in_tnobj, s.anonfunc_id, []), sym)
         end
     end
diff --git a/base/serialize.jl b/base/serialize.jl
index 70f1203d59..caad7e3a0a 100644
--- a/base/serialize.jl
+++ b/base/serialize.jl
@@ -438,8 +438,7 @@ end

 function serialize(s::AbstractSerializer, g::GlobalRef)
     if (g.mod === __deserialized_types__ ) ||
-        (g.mod === Main && isdefined(g.mod, g.name) && isconst(g.mod, g.name))
-
+            (g.mod === Main && isdefined(g.mod, g.name) && isconst(g.mod, g.name))
         v = getfield(g.mod, g.name)
         unw = unwrap_unionall(v)
         if isa(unw,DataType) && v === unw.name.wrapper && should_send_whole_type(s, unw)
```
rather than having a blank line